app/views/layouts/_nav.html.erb

```diff
@@ -3,6 +3,8 @@
     <%= nav_item( title: "Service performance", url: support_service_performance_path )  %>
     <%= nav_item( title: "Responsible bodies", url: support_responsible_bodies_path )  %>
     <%= nav_item( title: "Background jobs", url: support_sidekiq_admin_path, html_options: {target: '_blank'} )  %>
+  <%- elsif @user&.is_computacenter? %>
+    <%= nav_item( title: 'Home', url: '#') %>
```

Comment:
This is getting quite messy now. I wonder whether:

- `ApplicationController` should have an empty array of nav items
- `_nav.html.erb` should just show or display the nav items depending on whether the array is empty or not
- the Base controllers (or more specific controllers) are responsible for pre-populating the nav items per action?

Reply:
....or we could just have a different partial for each kind of user?

I'll log a Tech Debt card on Trello to improve this